packages/components/src/styles/components/time.scss

```
@@ -10,4 +10,9 @@
 .hds-time--range {
   display: inline-flex; // remove excess space between time range elements
   gap: 2px; // add a bit of space around en dash so it doesn't touch some number characters
+}
+
+// Note: The time-wrapper is only present when time is wrapped by a tooltip button
+.hds-time-wrapper {
```

[nit] if you want to be even more specific
```suggestion
.hds-time-wrapper .hds-time {
```